util/pace/util/local_comm.py

```diff
@@ -24,14 +25,29 @@ def wait(self):
         return self._result()
 
 
+class SplitState(Enum):
+    NO_SPLIT = 0
+    SPLITTING = 1
+    ALREADY_SPLIT = 2
```

Comment:
I don't understand why we need this if only no split is used?

Reply:
Vestigial code. Deleted.